public/app/AppWrapper.tsx

```diff
@@ -91,29 +93,32 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
           <ErrorBoundaryAlert style="page">
             <ConfigContext.Provider value={config}>
               <ThemeProvider>
-                <ModalsProvider>
-                  <GlobalStyles />
-                  <div className="grafana-app">
-                    <Router history={locationService.getHistory()}>
-                      {newNavigationEnabled ? <NavBarNext /> : <NavBar />}
-                      <main className="main-view">
-                        {pageBanners.map((Banner, index) => (
-                          <Banner key={index.toString()} />
-                        ))}
-
-                        <AngularRoot />
-                        <AppNotificationList />
-                        <SearchWrapper />
-                        {ready && this.renderRoutes()}
-                        {bodyRenderHooks.map((Hook, index) => (
-                          <Hook key={index.toString()} />
-                        ))}
-                      </main>
-                    </Router>
-                  </div>
-                  <LiveConnectionWarning />
-                  <ModalRoot />
-                </ModalsProvider>
+                <KBarProvider actions={[]} options={{ enableHistory: true }}>
+                  <ModalsProvider>
+                    <GlobalStyles />
+                    {config.featureToggles.commandPalette && <CommandPalette />}
```

Comment:
rendering this component here cause causes the command palette to render also in the login page. non authenticated users cannot run any actions, but pressing cmd+k still shows the KBar:
<img width="1019" alt="Screenshot 2022-04-08 at 13 34 20" src="https://user-images.githubusercontent.com/1170767/162436464-3c9f13c6-9e4c-4d95-b3bc-bc586152e7ed.png">


also, on every page load (or when signing out) we are calling `getDashboardNav` (which uses `/api/search`). given unauthenticated users cannot perform such request, this notification is shown at every page load:
<img width="1015" alt="Screenshot 2022-04-08 at 13 36 49" src="https://user-images.githubusercontent.com/1170767/162436868-89aa669a-cf8d-4ed8-825b-3053bc640d7c.png">